appcli/commands/task_cli.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# # -*- coding: utf-8 -*-
+
+"""
+Commands for application tasks.
+________________________________________________________________________________
+
+Created by brightSPARK Labs
+www.brightsparklabs.com
+"""
+
+# standard libraries
+import sys
+
+# vendor libraries
+import click
+
+# local libraries
+from appcli.logger import logger
+from appcli.models.configuration import Configuration
+
+# ------------------------------------------------------------------------------
+# CLASSES
+# ------------------------------------------------------------------------------
+
+
+class TaskCli:
+
+    # --------------------------------------------------------------------------
+    # CONSTRUCTOR
+    # --------------------------------------------------------------------------
+
+    def __init__(self, configuration: Configuration):
+        self.cli_configuration: Configuration = configuration
+        self.orchestrator = configuration.orchestrator
+
+        # ----------------------------------------------------------------------
+        # PUBLIC METHODS
+        # ----------------------------------------------------------------------
+
+        @click.group(
+            invoke_without_command=True,
+            help="Commands for application tasks.",
+        )
+        @click.pass_context
+        def task(ctx):
+            if ctx.invoked_subcommand is not None:
+                # subcommand provided
+                return
+
+            click.echo(ctx.get_help())
```

Comment:
Do we actually need this? I thought a group auto prints out help if the subcommand is not supplied?
If so, then just use `pass` for the body.

Reply:
Yes it's required. All our `@click.group` implement this same method, which is why all our groups 'auto-print' the help.